frontend/js/angular/controller.js

```diff
@@ -762,105 +778,22 @@ RodiApp.controller('RodiCtrl', ['$scope', 'RodiSrv', '$window', '$filter', '$loc
 
                         $scope.bLoadingTabel = false;
 
-                        // angular.element(document).ready(function () {
-                        //     jQuery('#tabCountries').DataTable(
-                        //         {
-                        //             ordering: true,
-                        //             paging: false,
-                        //             searching: false,
-                        //             info: false
-                        //         }
-                        //     );
-                        // });
+                        $scope.orderArrayByProperty = function(property, descending, field)
```

Comment:
`descending` is misleading in term of name as its value can be `up` (?) or `down`.

It would make more sense if it was named like `sortOrder` or `direction` to convey the meaning of its content.

---

Also, what is the difference between `property` and `field`?

Reply:
@oncletom the variable name is not a problem, I can change it...
Property and filed can be the same value: form me "property" is exactly the object field name and "field" is only a short name to use in ng-if expression.